### DISCUSSION TITLE
Add an "acquisition method" field to Fluxes Relation ?

### DISCUSSION THREAD

**Discussion by jfcjm** [dbarzin/mercator#107] [Ideas]
Bonsoir,
(Pour le moment ANSSI=>plutôt orienté francophonie, je continue donc en français :)
 
Il est actuellement possible d'indiquer si un échange entre deux éléments est chiffré ou non. 

Serait-il possible d'ajouter un champ permettant d'indiquer de quelle manière s'effectue l'échange :par exemple  "saisie manuelle", "transfert de fichier",  "partage de fichier", "api", "réplication de base de donnée" ?

Ce champ n'est pas indiqué dans la doc ANSSI mais sa présence permettrait de mieux préciser le contexte de sécurité de l'échange considéré.

J.-M.

**Comment by dbarzin** [comment]
Bonne idée. Dans un premier temps, cette information peut-être renseignée dans le champ description du flux.
Cette demande d'amélioration va être ajoutée à la feuille de route.
Merci,
Didier

**Comment by jfcjm** [comment]
OK, merci. Je vais créer une issue (un peu pour voir si ça fonctionne)...
